src/components/Home/Home.js

```diff
@@ -10,6 +15,8 @@ class Home extends Component {
     super(props)
     this.state = {
       authMessage: '',
+      activeEl: 'home',
```

Comment:
What is `El` mean? I think it must be more descriptive. 🤔

Reply:
`activeEl` is supposed to mean "activeElement". But that's not really a good name. I'll go ahead and call it `activeComponent`.